connection.go

```diff
@@ -293,6 +300,17 @@ type SslOpts struct {
 	Ciphers string
 }
 
+// Copy returns the copy of an Opts object.
```

Comment:
I suggest to use `Clone` instead of `Copy`. It may be more obvious.

https://pkg.go.dev/golang.org/x/exp/slices#Clone
https://pkg.go.dev/strings#Clone

Reply:
Thank you, changed